lighthouse-core/scripts/i18n/collect-strings.js

```diff
@@ -10,8 +10,16 @@
 
 const fs = require('fs');
 const path = require('path');
+const esprima = require('esprima');
 
 const LH_ROOT = path.join(__dirname, '../../../');
+const UISTRINGS_REGEX = /UIStrings = (.|\s)*?\};\n/gim;
+
+/**
+ * @typedef ICUMessage
```

Comment:
`ICUMessage.message` is a little unfortunate (e.g. destructuring like `const [key, msg] of Object.entries(strings)` below would be weird because then you'd need to say `msg.message`) , but I don't have anything better. `ICUMessageDefn` :) Not sure if there are any other terms of art

Reply:
`*Defn` it is!